addons/website_event/tests/test_website_event.py

```diff
@@ -72,6 +72,23 @@ def test_website_event_pages_seo(self):
         self.assertEqual(specific_view.website_meta_title, "Hello, world!")
         self.assertEqual(event.website_meta_title, False)
 
+    def test_website_event_submenu_align_breadcrumb(self):
+        self.env.ref('base.user_admin').write({
+            'name': 'Mitchell Admin',
+            'street': '215 Vine St',
+            'phone': '+1 555-555-5555',
+            'city': 'Scranton',
+            'zip': '18503',
+            'country_id': self.env.ref('base.us').id,
+            'state_id': self.env.ref('base.state_us_39').id,
+        })
```

Comment:
why need to change the admin demo data?
I think your tour will work without this data also